app/shared/styles.js

```diff
@@ -77,4 +77,12 @@ const styles = StyleSheet.create({
   }
 });
 
+// font family names:
```

Comment:
I added custom fonts via Xcode and in order to use them correctly, their exact names must be known without a single typo. So I left them here as reference.